### ISSUE TITLE
Cannot use Postman without internet connection - NO ACCESS TO COLLECTIONS

### ISSUE THREAD

**Issue by todduk** [a85/POSTMan-Chrome-Extension#4137]
I use Postman in a VM which has not had internet access for more than a month. Today when I load Postman I am shown a sign in screen with no possibly to do anything constructive. 

I have no access to my previous work, and clicking 'Cancel and take me to signed in account' does nothing.

This is absolutely unacceptable.

**Comment by edwardgalligan** [comment]
See also #3819

**Comment by sdnts** [comment]
@todduk, Postman does not restrict you from using the app without an account/internet connection. However, if you were signed in to an account earlier, and the access token used to authenticate you with our servers has expired, you have to sign back in to confirm your identity. You can still sign out completely and continue using the app normally, but your collections will not be visible to you.

This is a very common security measure taken to ensure your account is not misused.

If you weren't signed in in the first place, this should not happen. Please let me know if this is the case, and I'll reopen this :)

**Comment by jaxgibbmq** [comment]
I have this same issue - enterprise VPN doesn't allow login, so I use it offline. Suddenly, I can't get past the login screen & can't use the menus (check for update, etc). Win7, v5.5.3

**Comment by judgej** [comment]
Right at the bottom of the login window, in light grey and an 8px font, is a link to take you to the offline version of the app. You obviously are not meant to us it, but it's there, and you can use it to bypass the login page.

**Comment by SaltScratch** [comment]
@judgej I cannot see the link you mentioned. The only link I see at the bottom of the page is the 'Cancel and take me to signed in account' link which just reloads the login page

**Comment by judgej** [comment]
It was light grey on a lighter grey background, and I did not notice it at first, so wondering if it only appears after you have cancelled login a few times?

**Comment by Link631** [comment]
How can I install Postman Client offline? When I run the 64 bit setup.exe on my mashine, I get an error that I can not connect to the authentication service. Because of that I am not even able to install postman.

**Comment by OCram85** [comment]
Same issue here with Postman 7.14.0 x64. 

My current workaround:

- Set the focus on the retry button *(use the tabulator key)*
- Press enter
- Be quick with hitting the link below the retry button

💣 💣 💣 💣 💣 💣 💥

**Comment by elpinguinofrio** [comment]
in 2023 you can't use it without some sort of the account, see you https://github.com/postmanlabs/postman-app-support/issues/12145. just google "open source postman alternative"

**Comment by Arkemlar** [comment]
Recently I had to do some simple tasks in Postman while sitting on local network that had no internet access. And I was able to work only with what was opened in tabs and cached. I was not able to pick request from Collections. Opening console bottom window also failed. And sometimes I could not send request that has been open right in front of my eyes! Every mentioned action just hanged on Postman in loading state, eventually showing showing error screen. After that there were nothing to do but restart this application.
Looks like Postman needs internet connection for every sneeze. I really don't understand that. There are no need for any network to open collection item, to open console window, and so on. Everything worked fine without network a couple years ago.

Postman has been an excellent tool for a decade and I loved it. But in recent years it degraded too much in my POV. It became slow, resource greedy, and as I discovered today - absolutely unusable without internet connections. 
This is not acceptable. I was thinking about alternatives before, but after being pissed off by this issue, I will forget about Postman for sure.
